README.md

```diff
@@ -60,7 +61,8 @@ You can view the list of [TiKV Adopters](https://tikv.org/adopters/).
 
 ## TiKV software stack
 
-![The TiKV software stack](images/tikv_stack.png)
+![The TiKV software stack](images/tikv_stack.png#gh-light-mode-only)
```

Comment:
How about using the same white background images?

Reply:
Updated it to use a single image with white background.